src/bin/zmpython

```diff
@@ -6,4 +6,6 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
 export TMP=/opt/zextras/data/tmp
-exec /opt/zextras/bin/zmjava -Djava.io.tmpdir=/opt/zextras/data/tmp -Dpython.cachedir.skip=true org.python.util.jython "$@"
+exec /opt/zextras/bin/zmjava \
+-classpath "/opt/zextras/jython/jars/*:/opt/zextras/mailbox/jars/*:/opt/zextras/conf" \
```

Comment:
I would not create a dedicated `/opt/zextras/mailbox/` folder at least not in `/opt/zextras` where a mailboxd folder already exists. This could lead to confusion and would be better to have jars under already present path.

Reply:
Where do you recommend putting these jars?

Reply:
@M0Rf30  I updated the files to use /opt/zextras/mailboxd/jars